catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/metacard/RegistryObjectMetacardType.java

```diff
@@ -69,8 +71,23 @@
 
     public static final String PUBLISHED_LOCATIONS = "published-locations";
 
+    public static final String LAST_PUBLISHED = "last-published";
+
     public static final String REGISTRY_BASE_URL = "registry-base-url";
 
+    public static final String REGISTRY_LOCAL_NODE = "registry-local-node";
+
+    public static final Set<String> TRANSIENT_ATTRIBUTES;
+
+    static {
+        final Set<String> transientAttributes = new HashSet<>();
```

Comment:
Another option is adding Guava as a dependency (its packages are exported) and using `ImmutableSet.of(...)`. But this is fine as it stands.